src/main/java/ch/njol/skript/aliases/Aliases.java

```diff
@@ -403,7 +403,10 @@ private static void loadMissingAliases() {
 		for (Material material : Material.values()) {
 			if (!material.isLegacy() && !provider.hasAliasForMaterial(material)) {
 				NamespacedKey key = material.getKey();
-				String name = key.getKey().replace("_", " ");
+				String name = (key.getNamespace().equals(NamespacedKey.MINECRAFT) ? key.getKey() : key.toString())
+								  .replace("_", " ");
+				// mod:an_item → mod:an item
+				// minecraft:dirt → dirt
```

Comment:
What do you think about `mod's an item` instead of `mod:an item`
For example: `tconstruct's seared stone bricks`

Reply:
sounds great👍

Reply:
that's a good idea!